tests/integration/test_fim/test_registry/test_ignore/test_ignore_registry.py

```diff
@@ -8,66 +8,45 @@
+from wazuh_testing import global_parameters
+from win32con import REG_SZ
 
-import win32api
-import win32con
-
-
-# All tests in this module apply to windows only
 pytestmark = [pytest.mark.win32, pytest.mark.tier(level=1)]
 
 # variables
-
-test_data_path = os.path.join(os.path.dirname(os.path.realpath(__file__)), 'data')
-configurations_path = os.path.join(test_data_path, 'wazuh_conf_win32_ignore_registry.yaml')
-
-reg_handlers = list()
-
 key = "HKEY_LOCAL_MACHINE"
+subkey_1 = "SOFTWARE\\test_key"
+subkey_2 = "SOFTWARE\\Classes\\test_key"
+ignore_key = "key_ignore"
+ignore_regex = "ignored_key$"
+ignore_value = "value_ignored"
+ignore_value_regex = "ignored_value$"
 
-regs = [os.path.join('SOFTWARE', 'test'), os.path.join('SOFTWARE', 'test2')]
-sub_keys = [(os.path.join(regs[0], 'testreg32'), '32'),
-            (os.path.join(regs[0], 'testreg64'), '64'),
-            (os.path.join(regs[1], 'testregboth_32'), "both"),
-            (os.path.join(regs[1], 'testregboth_64'), "both")]
-
+test_data_path = os.path.join(os.path.dirname(os.path.realpath(__file__)), 'data')
 wazuh_log_monitor = FileMonitor(LOG_FILE_PATH)
+test_regs = [os.path.join(key, subkey_1), os.path.join(key, subkey_2)]
 
-# configurations
-
-configurations = load_wazuh_configurations(configurations_path, __name__)
-
-
-def extra_configuration_before_yield():
-    """ Initialize registries for this test """
-    for reg in regs:
-        reg_handlers.append(create_registry(registry_parser[key], reg, win32con.KEY_WOW64_32KEY))
-        reg_handlers.append(create_registry(registry_parser[key], reg, win32con.KEY_WOW64_64KEY))
-
-    for keys in sub_keys:
-        if (keys[1] == "both"):
-            create_registry(registry_parser[key], keys[0], win32con.KEY_WOW64_32KEY)
-            create_registry(registry_parser[key], keys[0], win32con.KEY_WOW64_64KEY)
-        else:
-            arch = win32con.KEY_WOW64_32KEY if keys[1] == '32' else win32con.KEY_WOW64_64KEY
-            create_registry(registry_parser[key], keys[0], arch)
-
-
-def extra_configuration_after_yield():
-    for keys in sub_keys:
-        if (keys[1] == "both"):
-            delete_registry(registry_parser[key], keys[0], win32con.KEY_WOW64_32KEY)
-            delete_registry(registry_parser[key], keys[0], win32con.KEY_WOW64_64KEY)
-        else:
-            arch = win32con.KEY_WOW64_32KEY if keys[1] == '32' else win32con.KEY_WOW64_64KEY
-            delete_registry(registry_parser[key], keys[0], arch)
+reg1, reg2 = test_regs
 
+# Configurations
+conf_params = {'WINDOWS_REGISTRY_1':reg1,
+               'WINDOWS_REGISTRY_2':reg2,
+               'REGISTRY_IGNORE_1': os.path.join(reg1, ignore_key),
+               'REGISTRY_IGNORE_2' : os.path.join(reg2, ignore_key),
+               'REGISTRY_IGNORE_REGEX': ignore_regex,
+               'VALUE_IGNORE_1' : os.path.join(reg1, ignore_value),
+               'VALUE_IGNORE_2' : os.path.join(reg2, ignore_value),
+               'VALUE_IGNORE_REGEX' : ignore_value_regex
```

Comment:
Whitespace before :.
```suggestion
               'VALUE_IGNORE_1': os.path.join(reg1, ignore_value),
               'VALUE_IGNORE_2': os.path.join(reg2, ignore_value),
               'VALUE_IGNORE_REGEX': ignore_value_regex
```